spring-cloud-dataflow-docs/src/main/asciidoc/getting-started-cloudfoundry.adoc

```diff
@@ -97,7 +97,7 @@ applications:
   path: <PATH TO THE DOWNLOADED SKIPPER SERVER UBER-JAR>
   env:
     SPRING_APPLICATION_NAME: skipper-server
-    SPRING_CLOUD_SKIPPER_SERVER_ENABLE_LOCAL_PLATFORM: false
+    SPRING_APPLICATION_JSON: '{ "spring": { "cloud": { "skipper": { "server": { "enableLocalPlatform": "false"} } } } }'
```

Comment:
I don't think this prpoperty works whether in this format or as flat env-var. That's a bug outside of this change, in skipper perhaps.